assets/js/hooks/session.js

```diff
@@ -1447,12 +1468,44 @@ const Session = {
 
   jumpToLine(file, line) {
     const [_filename, cellId] = file.split("#cell:");
-
     this.setFocusedEl(cellId, { scroll: false });
     this.setInsertMode(true);
 
     globalPubsub.broadcast(`cells:${cellId}`, { type: "jump_to_line", line });
   },
+
+  saveNavigationHistory({ cellId, line, column }) {
+    if (cellId === null || line === null || column === null) return;
```

Comment:
Do we ever receive such event?

Reply:
Now we don't receive anymore, I can drop this one